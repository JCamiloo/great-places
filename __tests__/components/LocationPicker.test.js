import React from "react";
import LocationPicker from "../../components/LocationPicker";
import renderer from "react-test-renderer";

describe("LocationPicker component tests", () => {
  test("LocationPicker component renders correctly", () => {
    const tree = renderer
      .create(<LocationPicker navigation={{ getParam: jest.fn() }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
