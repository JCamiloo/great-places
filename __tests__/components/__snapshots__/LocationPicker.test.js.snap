// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationPicker component tests LocationPicker component renders correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 15,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "borderColor": "#ccc",
        "borderWidth": 1,
        "height": 150,
        "justifyContent": "center",
        "marginBottom": 10,
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text>
      No location chosen yet
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-around",
        "width": "100%",
      }
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#fc9208",
              },
            ]
          }
        >
          Get user location
        </Text>
      </View>
    </View>
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#fc9208",
              },
            ]
          }
        >
          Pick on Map
        </Text>
      </View>
    </View>
  </View>
</View>
`;
